open-reac/src/main/java/com/powsybl/openreac/parameters/input/OpenReacParameters.java

```diff
@@ -58,6 +58,46 @@ public class OpenReacParameters {
 
     private ReactiveSlackBusesMode reactiveSlackBusesMode = ReactiveSlackBusesMode.NO_GENERATION;
 
+    private static final String ALPHA_COEFFICIENT_KEY = "coeff_alpha";
```

Comment:
This parameter is sometimes named "alphaCoefficient" and sometimes "coeff_alpha" which sounds French to me... Can't we use "alpha_coeff" instead to be consistent?

Beside that, is "alphaCoefficient" a well-known name for users?

Reply:
Ok good point, let's not change `coeff_alpha` in this PR. We can do that later.

And giving it another name in the Java world is a good idea too. The variable is a boolean so naming it `actiovePowerVariationRate' is weird though... I suggest `activePowerVariationFunction` but I am not 100% convinced either. You pick :)

Reply:
After discussion, the `alphaCoefficient` is not a boolean (my mistake). So let's keep `activePowerVariationRate`